application/views/themeOptions/options_core.php

```diff
@@ -126,7 +126,7 @@
                                         $sParentLabelOption = $options[$sParentOption];
                                         $options['inherit'] = gT("Inherit") . ' [' . gT($sParentLabelOption) . ']';
                                     } else {
-                                        $sParentOption = $sParentOption !== '' ? gT($sParentOption) : $sParentOption;
+                                        $sParentOption = isset($options[$sParentOption]) ? gT($options[$sParentOption]) : $sParentOption;
```

Comment:
usage of empty maybe ? here you seems you can go to gT("") ?

And previous system seems to send '' as value